build_tools/py/vpip.py

```diff
@@ -69,6 +69,8 @@ def get_unix_pip_env(venv, venv_source, execroot):
         env["F77"] = env["F90"] = os.path.join(os.getcwd(), ARGS.fortran_compiler)
     env["AR"] = os.path.join(os.getcwd(), ARGS.archiver)
     env["CC"] = os.path.join(os.getcwd(), ARGS.compiler_executable)
```

Comment:
Would it be reasonable to do the simpler thing of adding `env["MPICC"] = env["CC"]` here and not doing any of the other changes?

Reply:
My only concern is that there might be build scripts that assume that the presence of the environment variable MPICC that it was a valid mpi compiler wrapper and hence try to enable mpi features even if the compiler doesn't work as an MPI compiler. 

The reality is though that I am setting it to the same value as `CC` in `_add_vpip_compiler_args` and its working for us so I guess its good enough? If we come across any packages this causes an issue we can revisit this with a similar approach to the fortran feature (we're already doing this to select the mpi compiler wrapper for normal cc_* targets).

Reply:
Right, my proposal and this change seem totally equivalent, which is why I asked.